kuma/search/utils.py

```diff
@@ -47,3 +55,16 @@ def clean_params(self, params):
             if default not in ('', None):
                 clean_params[param] = default
         return clean_params
+
+
+def search_exception_handler(exc):
+    # Call REST framework's default exception handler first,
+    # to get the standard error response.
+    response = exception_handler(exc)
+
+    if (response is None and
+            isinstance(exc, elasticsearch.ElasticsearchException)):
+        return Response({'error': SEARCH_DOWN_DETAIL},
+                        status=status.HTTP_200_OK)
```

Comment:
Maybe a note here that 503 would be the correct response but due to the proxy, blah blah blah.